test/test-temp-directory.js

```diff
@@ -36,6 +47,18 @@ test('tempDirectory.create:', function (t) {
   });
 });
 
+test('tempDirectory.create --tmpDir:', function (t) {
+  tempDirectory.create(contextTmpDir, function (e, ctx) {
+    t.error(e);
+    t.equals(ctx.path, 'thisisatest/test-module', 'context should now have a path');
```

Comment:
@MylesBorins `this line should still use uuid to generate a nonce.`
We will have to use some regex to match here if you want to use uuid.